cmd/client.go

```diff
@@ -122,6 +122,8 @@ user-defined plans.
 			if err := client.RunExamplesForService(apiClient, serviceName); err != nil {
 				log.Fatalf("Error executing examples: %v", err)
 			}
+
+			log.Println("Success")
```

Comment:
This is just a long-standing deficiency. No more needing to call `$?` to double-check the exit-status.